incl/footer.html

```diff
@@ -12,6 +12,13 @@
     <div id="block-aggregator-feed-1" class="clear-block block block-aggregator">
       <h2 class="title">Recent news</h2>
       <div style="background-color: blanchedalmond; padding-left: 5px;" class="content">
+        <p>October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'.
+          Renaming all existing resources is a lot of work, it will be a slow, gradual transition.
+          During this transition period, please feel free to use both names, for example
+          by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".
```

Comment:
That's not the official position of the Coq team, which is instead that as long as the renaming hasn't happened, people should still refer to Coq as Coq: https://x.com/CoqLang/status/1746558655758041477

That being said, people are free not to respect this (and in practice, some people have already started referring to Coq as Rocq).

In published papers, my personal recommendation would be to add a footnote on the first mention of Coq to say that there is a planned renaming to "the Rocq prover".

Reply:
Thanks for the feedback!

>  I would make this piece of news briefer, though.

Could you suggest a wording that you like, here in the comments?

Reply:
@Zimmi48 I made a new iteration that reads as follows (and removes all other changes, as you suggested):

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. The rename is currently in preparation, and will hopefully happen sometime in 2024.